src/components/MessageInput/CooldownTimer.tsx

```diff
@@ -4,20 +4,24 @@ export type CooldownTimerProps = {
   cooldownInterval: number;
   setCooldownRemaining: React.Dispatch<React.SetStateAction<number | undefined>>;
```

Comment:
I think we don't need this prop anymore, right?

Reply:
Removing it would be a breaking change as the interface is given also for the custom variations of the timer:

![image](https://github.com/GetStream/stream-chat-react/assets/32706194/affe0bf3-09bd-4b24-a1ff-e7ddd6049125)

Reply:
ok, I see... it is messed up :)